plop-templates/reset-db.ts.hbs

```diff
@@ -0,0 +1,45 @@
+import process from "node:process"
+import chalk from "chalk"
+import { Client } from "pg"
+import { getEnv } from "env"
+import { migrate } from "./migrate-db"
+import { getConnectionStringFromEnv } from "pg-connection-from-env"
+
+export const reset = async () => {
+  const { DATABASE_URL } = getEnv()
+
+  const seam_db = new Client({
+    connectionString: DATABASE_URL,
+  })
+  const postgres_db = new Client({
+    connectionString: getConnectionStringFromEnv(
+      DATABASE_URL,
+      "postgres"
+    ),
```

Comment:
i don't think this works https://github.com/seamapi/pg-connection-from-env/blob/01571d0ccce2a1bc01adce08f35fede436e539ac/index.ts#L73